Module4.1_Intro_to_Data_Structures_and_Algos/activities/Act1_TimeAndSpaceComplexity/2.md

```diff
@@ -53,7 +53,8 @@ Hence, we predict that `three()` grows in **constant time**.
 
 As shown in the image above, for any number of elements in `value`, `valueSum()` always takes about 0.001 milliseconds to run. 
 
-![alt](data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAUEAAACdCAMAAAAdWzrjAAAAnFBMVEX////09PT/AABTU1Onp6e5ubno6Oj/e3r/7Ov/eXj/DgX/Skn/R0X/p6f/qanR0dEAAAAdHR3b29svLy+RkZH4+PhcXFxNTU3/zs0mJiaAgIBvb2+dnZ09PT3Jycm/v797SEj2l5f/u7v/jIv/U1KIiIilpaX/mZn/3dz/8/Lm7e3/OjhtbW1iYmJDQ0NYWFgTExPtvLthTU2jOzq4mG8uAAACb0lEQVR4nO3Ya2/SUByA8dNB62UblNJSuc7LmE7FTf3+382yqa8ODcmDlibPrwkJyf/FyVPOaUoIkiRJ0j+3WWRdL6Hn0tqCTDqwIGNByoKUBSkLUhakLEhZkLIgZUHKgpQFKQtSFqQsSFmQsiBlQcqClAUpC1IWpCxIWZCyIGVByoKUBSkLUhakLEhZkLIgZUHKgpQFKQtSFqQsSFmQsiBlQcqClAUpC8aMb/bGyeaIWQvGlLNZPpuVxeCIWQvGVZOq+Uyaa1yE599iMY5OWjAumSTNZp6EZb7Ll+u6DuHh226eRCYtGPdccBqWq5DmIcyztNnR5SIyacG4PwW/pk2iUE2z7eN6PV1GJi0Y97fg5nfBctscju7i4yWP+4LzsGsK1qGaF0W+SaaryKQF46pFU7BYhFURxmXzLQvjh/UwNmlByoKUBSkLUhakLEhZkLIgZUHKgpQFKQtSFqQsSFmQsiBlQcqClAUpC1IWpCxIWZCyIGVByoKUBSkLUhakLEhZkLIglQ4+d72Enkt/fH+z97LVfdfLPGPpz4uTeH3YxdVlm9sXrT6Mnly/77rUIenbd6PrvdHHV60ur9p8Oc19aHHbdalDzuIcvG8/Q56OmU9dL/IQn8WUBSkLUhakLEhZkLIgZUHKgpQFKQtSFqQsSFmQsiBlQcqClAUpC1IWpCxIWZCyIGVByoKUBSkLUhakLEhZkLIgZUHKgpQFKQtSFqTSQVElfVB1XeqQzXLQC3XZdamDsrthH9zddB1KkiRJ+n/O9hWsH4bben2+72B9UE6TLPcfH6BchVAXXa+iz8phCDMLAhakLCjpNH4Bn7sl7NukZi4AAAAASUVORK5CYII=)
+![www.askiitians.comiit-jee-algebraset-relations-functionsimagesgraph-of-a-constant-function.jpg](/Users/brindapuri/Downloads/www.askiitians.comiit-jee-algebraset-relations-functionsimagesgraph-of-a-constant-function.jpg.jpg)
```

Comment:
This image is local, if you don't have a way to make it a non local image send it to me and I can fix it!